tests/fltest_screen.fish

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env fish
+# -*-  mode:fish; tab-width:4  -*-
+
+function fltest_screen
+
+    echo "Context: STY var is set to 'fishline_test'"
```

Comment:
hah, thanks!  I actually have no idea what i'm doing when it comes to writing tests for this kind of thing, but that makes sense.  I'll correct.